text/0000-buildpack-versions.md

```diff
@@ -0,0 +1,56 @@
+# Meta
+[meta]: #meta
+- **Name:** Flexible Buildpack Versions
+- **Start Date:** 2020-03-11
+- **Supersedes:** N/A
+
+# Summary
+[summary]: #summary
+
+The buildpacks project **does not** want to enforce rules or format requirements for buildpack versions. This RFC explains that buildpack authors can use any pattern (ie. semantic versioning) when deciding how to release versions of buildpacks.
+
+# Motivation
+[motivation]: #motivation
+
+There aren't specific guidelines in the buildpack documentation for buildpack authors that explain how versioning works for buildpacks. It should be explicitly stated that the API supports any version format.
+
+Therefore, buildpack authors have clear documentation that they can use any versioning system that works for their development workflow.
+
+# What it is
+[what-it-is]: #what-it-is
+
+Buildpack authors can version buildpacks how they want. This requires no additional code, but instead clarification around how versions are referred to.
```

Comment:
it might be worth calling out that this is the status quo, and what you are proposing is to codify that in the spec.